src/vanilla/ClientModelExtensions.cs

```diff
@@ -853,53 +853,27 @@ public static string ConstructMapper(this IModelType type, string serializedName
                     builder.AppendLine("className: '{0}',", composite.Name)
                            .AppendLine("modelProperties: {").Indent();
                     var composedPropertyList = new List<Property>(composite.ComposedProperties);
+                    PropertyInfo itemName = composite.GetType().GetProperty("ItemName");
+                    string itemNameValue = (string)itemName?.GetValue(composite);
+                    PropertyInfo nextLinkName = composite.GetType().GetProperty("NextLinkName");

```

Comment:
These line changes are just hoisting pointless repetition out of the loop?

Reply:
Correct.